src/calling-convention.adoc

```diff
@@ -0,0 +1,29 @@
+[appendix]
+== Calling Convention (Not authoritative - Placeholder Only)
```

Comment:
Title should be changed to reflect this is the calling convention (placeholder) only the vector state.

Reply:
I've renamed it "Calling Convection for Vector State...."